src/models/event.rs

```diff
@@ -16,6 +16,17 @@ pub struct Event {
 }
 
 impl Event {
+    pub fn new_test() -> Event {
```

Comment:
You should not put test code functions in test structs

Reply:
I know, I just put it there because I didn't know where to put test utils. Instead, I rewrote it into a plain `new` helper function, is that fine?

Reply:
See my other comment about default trait